nursery/wmi-get-antivirus.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: wmi get antivirus
+    namespace: collection/anti-virus
+    authors:
+      - kevross33/Kevin Ross
+    scopes:
+      static: function
+      dynamic: span of calls
+    att&ck:
+      - Discovery::Windows Management Instrumentation [T1047]
+    examples:
+      - f5fca1b178af87bd48c7ea9e3f2c957b
+  features:
+    - and:
+      - string: /root\\securitycenter/i
+      - string: /antivirusproduct/i
```

Comment:
This is a little broad - is there anything else we could add to reduce the risk of FPs (e.g. `api`, `match` features)?